pikaptcha/console.py

```diff
@@ -148,6 +148,9 @@ def entry():
                 ulist.close()
         for x in range(0,args.count):
             print("Making account #" + str(x+1))
+            if (((x+1)%6 = 0)):
```

Comment:
Well this is embarrasing ,i don't know python.From the 200+ accounts i created i definitely run the command for 5 accounts every 5 minutes or maybe more ,lets say 10 minutes,definitely not 1 hour.